components/org.wso2.carbon.identity.application.authenticator.oauth2/src/main/java/org.wso2.carbon.identity.application.authenticator.oauth2/Oauth2GenericAuthenticator.java

```diff
@@ -487,10 +486,9 @@ public String getContextIdentifier(HttpServletRequest request) {
         }
     }
 
-    protected String generateState() {
+    protected String getStateParameter(AuthenticationContext context) {
```

Comment:
Are we using this method in any places?

Reply:
getStateParameter() replaces the only usage in all places